src/Core/src/PublicAPI/net-windows/PublicAPI.Shipped.txt

```diff
@@ -1551,7 +1551,6 @@ Microsoft.Maui.Platform.ColorExtensions
 Microsoft.Maui.Platform.ContentPanel
 Microsoft.Maui.Platform.ContentPanel.ContentPanel() -> void
 Microsoft.Maui.Platform.ContentPanel.UpdateBackground(Microsoft.Maui.Graphics.Paint? background) -> void
-Microsoft.Maui.Platform.ContentPanel.UpdateBorderShape(Microsoft.Maui.Graphics.IShape! borderShape) -> void
```

Comment:
This is wrong no? we are adding and removing?

Reply:
Right, fixed.